ui/public/index.html

```diff
@@ -36,7 +36,7 @@
       name="viewport"
       content="minimum-scale=1, initial-scale=1, width=device-width"
     />
-    <title>React PDB</title>
+    <title>Tanagra UI</title>
```

Comment:
Is it redundant to have the title of a webpage include "UI"?

Reply:
That's fair. Removed it.